src/client/Client.cc

```diff
@@ -425,18 +436,23 @@ void Client::dump_cache(Formatter *f)
   if (f)
     f->open_array_section("cache");
 
-  if (root)
-    dump_inode(f, root, did, true);
+  if (inode_no) {
+      auto it = inode_map.find(vinodeno_t(inode_no, CEPH_NOSNAP));
+      if (it != inode_map.end())
+          dump_inode(f, it->second, did, true);
```

Comment:
Dump error message if not found?

Reply:
It returns an empty cache instead of an error message.

Reply:
Hmm, that's reasonable. :)